README.md

```diff
@@ -4,7 +4,7 @@ Hello Microverse project is to implement GitHub flow, a linter, and a profession
 
```

Comment:
- [ ] There is no linter setup for your repo, kindly ensure you have the linters.yml file in the .github/workflows folder.

Reply:
- [ ] Your branch should have a name of a feature or action (e.g. 'hello_microverse' or 'setup_repository') to provide more information on what it contains.

Reply:
- [OPTIONAL] Your PR title says Hello Microsoft instead of Hello Microverse, kindly fix it.